code/modules/events/wizard/aid.dm

```diff
@@ -11,11 +11,11 @@
 
 	for(var/mob/living/L in GLOB.mob_list) //Hey if a corgi has magic missle he should get the same benifit as anyone
 		if(L.mind && L.mind.spell_list.len != 0)
```

Comment:
While we're mucking about in the area, maybe we should change this to `if(L.mind && L.mind.spell_list.len)`?